### PR TITLE
[#978][Improvement] Provides a tool class to format CLI output content

### PR DESCRIPTION
**Pull request by yl09099**
### What changes were proposed in this pull request?

Provides a tool class to format CLI output content.

### Why are the changes needed?

Fix: #978

### Does this PR introduce _any_ user-facing change?

The CLI command display is more beautiful.Similar to：

image

### How was this patch tested?

Added UT.

### CODE REVIEW

File: cli/src/test/java/org/apache/uniffle/cli/CLIContentUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CLIContentUtilsTest {
+
+  @Test
+  public void testTableFormat() {
+
+    String expectStr = "+----------------------------------------"
```
Comment (slfan1989) [inline cli/src/test/java/org/apache/uniffle/cli/CLIContentUtilsTest.java:38]: Is there a better way? This looks difficult to read. @zuston Do you have a good suggestion?

Comment (slfan1989) [reply cli/src/test/java/org/apache/uniffle/cli/CLIContentUtilsTest.java:38]: Why do we need 5 rows? @yl09099

**Comment by yl09099** [comment]
@jerqi This error is not caused by me, take a hard look

File: cli/src/test/java/org/apache/uniffle/cli/CLIContentUtilsTest.java (hunk 1 above)
Comment (zuston) [reply cli/src/test/java/org/apache/uniffle/cli/CLIContentUtilsTest.java]: > Is there a better way? This looks difficult to read. @zuston Do you have a good suggestion?

Loading from the file?

**Comment by yl09099** [comment]
It has been modified, please review it @slfan1989 @zuston

**Comment by zuston** [comment]
> It has been modified, please review it @slfan1989 @zuston

Could this apply one cli command in this PR? I'm not sure we will use this in the current implementation currently.

Anyway, this looks good to me.

File: cli/src/test/java/org/apache/uniffle/cli/CLIContentUtilsTest.java (hunk 1 above)
Comment (slfan1989) [reply cli/src/test/java/org/apache/uniffle/cli/CLIContentUtilsTest.java]: The code looks better, let's wait for @zuston the suggestion.

**Comment by zuston** [comment]
Merged.

Thanks @yl09099

**Pull request merged**